calendar_resource/README.rst

```diff
@@ -0,0 +1,85 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
```

Comment:
Use new README by fragment system, copying https://github.com/OCA/maintainer-tools/tree/master/template/module/readme and editing or removing the needed sections. You can then run locally oca-gen-addon-readme if you have OCA/maintainer-tools installed and you want to pregenerate the README.rst file (preferred), or keep previous README.rst file.

Please check [OCA Doc](the https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#tasks-to-do-in-the-migration)

Reply:
Hey, thanks, I did the changes to include the fragmented system also I reviewed the guideline [v11migration](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0)